android-whitelist.json

```
@@ -113,7 +118,8 @@
         },
         {
             "group": "com\\.bugsnag",
-            "name": "bugsnag-android"
+            "name": "bugsnag-android",
+            "version": "4\\.15\\.0"
```

ya tenemos disponible la [v4.16.1 ](https://github.com/bugsnag/bugsnag-android/releases/tag/v4.16.1)
```suggestion
            "version": "4\\.16\\.1"
```

---

ojo que en Android todavia seguimos usando la 4.15. Esto no es la ultima version que exista, si no la ultima version que soportamos

---

Te parece q dejemos la ultima? no trae breaking changes, y así cuando la actualizamos no hace falta hacer este PR de nuevo ?

---

@abruno- compro si me haces el PR jajajajajja

---

resolucion: dejamos esta y LTA si se sube, hay que hacer PR aca